ckanext/publications_qld_theme/templates/package/comment_list.html

```diff
@@ -51,14 +55,17 @@ <h3>Reply to comment:</h3>
 
 {% macro comment_thread(thread, content_type, depth=0) %}
 
+    {% if thread.comments | length > 0 and notification_actions_enabled %}
+        {% snippet "snippets/content_item_notification_actions.html", following=following_content_item, content_type=content_type, thread_id=thread.id %}
+    {% endif %}
+
     {% for comment in thread.comments %}
     <div id="comment_{{ comment.id }}" class="comment-wrapper">
-
-        <div class="comment comment-{{ comment.state }}">
+        {% set user_can_manage = h.user_can_manage_comments(content_type, pkg_id) %}
```

Comment:
are we enabling commenting on publications?

Reply:
I believe the comment extension is not currently installed for Publications. It might be feasible in the future, though.